pkgs/by-name/ob/objfw/package.nix

```diff
@@ -25,7 +27,8 @@ clangStdenv.mkDerivation (finalAttrs: {
     automake
     autogen
     autoconf
-  ];
+  ]
+  ++ lib.optional openSslSupport openssl;
```

Comment:
There is also support for GnuTLS and MbedTLS, in case you want to support those, too.